api/Dockerfile

```diff
@@ -1,26 +1,27 @@
-FROM --platform=linux/amd64 python:3.11-slim as base
+FROM --platform=linux/amd64 python:3.11-slim AS base
 WORKDIR /code
 CMD ["/code/src/init.sh", "api"]
 EXPOSE 5000
 
 ENV PYTHONUNBUFFERED=1
 ENV PYTHONPATH=/code
 
-RUN pip install --upgrade pip && \
-    pip install poetry && \
-    poetry config virtualenvs.create false
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /bin/uv
 
 COPY pyproject.toml pyproject.toml
-COPY poetry.lock poetry.lock
+COPY uv.lock uv.lock
+
+RUN mkdir -p /.cache/uv && \
+    chown -R 1000:1000 /.cache/uv/
```

Comment:
not sure if this is  best practice, but uv needs read/write access to this folder. (even with --no-cache)

Reply:
Can undo this chown after running "uv sync". At least in the prod image